client/js/utils.js

```diff
@@ -17,12 +17,22 @@ module.exports = {
 	toggleNickEditor,
 	toggleNotificationMarkers,
 	requestIdleCallback,
+	isOpInChannel,
 };
 
 function resetHeight(element) {
 	element.style.height = element.style.minHeight;
 }
 
+// Given a channel element will determine if the lounge user is Op in that channel
+function isOpInChannel(channel) {
+	const channelID = channel.data("id");
+	const network = $("#sidebar .network").has(".chan[data-id=\"" + channelID + "\"]");
+	const ownNick = network.data("nick");
+	const isOP = channel.find(".users .user-mode.op .user[data-name=\"" + ownNick + "\"]").length;
```

Comment:
Use string interpolation and `CSS.escape` (`const escape = require("css.escape");` for `ownNick`.